vaadin-notification-flow-parent/vaadin-notification-flow-integration-tests/pom.xml

```
@@ -89,6 +89,12 @@
             <version>3.1.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-button-testbench</artifactId>
+            <version>23.1-SNAPSHOT</version>
```

Please use project version:

```suggestion
            <version>${project.version}</version>
```